Sources/CvGame.cpp

```diff
@@ -2673,24 +2673,6 @@ void CvGame::update()
 		}
 	}
 
-	if (isOption(GAMEOPTION_SIZE_MATTERS))
```

Comment:
Not sure what this is doing, however the static bool below ensures it will only ever run once in a session (i.e. you need to restart the game to run it again). Therefore it could either be entirely redundant, OR bugged.